server/bin/pbench-sync-satellite.sh

```diff
@@ -146,7 +146,7 @@ dest=$(getconf.py pbench-receive-dir-prefix pbench-server)-002
 let unpack_start_time=$(timestamp-seconds-since-epoch)
 
 for host in $hosts ;do
-    localdir=$dest/$remote_prefix::$host
```

Comment:
Let's not change the legacy behavior.  We just need the put shim to strip it off.

Reply:
Let me know if, based on this change of direction, I should resolve the [conversation above](https://github.com/distributed-system-analysis/pbench/pull/3187#discussion_r1082851464).

Reply:
Hey @webbnh the PR has changed, your reviews doesn't align with new approach so maybe we can resolve those.

Reply:
I'm confused: `pbench-sync-satellite.sh` is *NOT* changing to put the tarballs in the reception area? It is still putting them in the archive? How does `put-shim` figure out where to get them?

Reply:
> I'm confused: `pbench-sync-satellite.sh` is _NOT_ changing to put the tarballs in the reception area? It is still putting them in the archive? How does `put-shim` figure out where to get them?

That was implemented in PR #3181.

Reply:
D'oh - thanks!